libdeno/binding.cc

```diff
@@ -312,15 +339,35 @@ void Send(const v8::FunctionCallbackInfo<v8::Value>& args) {
   }
 }
 
+void Shared(const v8::FunctionCallbackInfo<v8::Value>& args) {
```

Comment:
I don't understand why this has to be a function.
Instead of making a function, you can just create the ArrayBuffer directly.
The AB can even be serialized in the snapshot (not it's contents, but the pointer to it).

Reply:
Changed to getter.